MoveCardAfterReview/ReadMe.md

```diff
@@ -6,6 +6,10 @@ If relevant card does not exist, a new one is created in the selected GitHub pro
 
 When PR contains one or more `Fixes #...` in the description, all referenced issues are moved. Otherwise, card for the standalone PR itself is moved.
 
+## Compatibility
```

Comment:
I forgot one thing, we need to add Prerequisities information about the token, the same way (with different link) as in the toggle lock branch:
https://github.com/SonarSource/gh-action-lt-backlog/tree/master/ToggleLockBranch#prerequisites

Reply:
done